rfc/rfc-0019-rfc-process.md

```diff
@@ -0,0 +1,112 @@
+
+- Fork the repo: <https://github.com/vmware-tanzu/cartographer>
+- Copy `rfc/rfc-0000-template.md` to `rfc/rfc-0000-my-feature.md` (where 'my-feature' is descriptive. don't assign an RFC number yet).
+- Fill in RFC. Any section can be marked as "N/A" if not applicable.
+- Submit a pull request. The pull request is the time to get review of the proposal from the larger community.
+- Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+
+Once a pull request is opened, the RFC is now in development and the following will happen:
+
+- It will be discussed in a future community meeting. Community meetings happen on a weekly cadence barring exceptions.
+- The team will discuss as much as possible in the RFC pull request directly. Any outside discussion will be summarized in the comment thread.
+- When deemed "ready", a team member will propose a "motion for final comment period (FCP)" along with a disposition of the outcome (merge, close, or postpone). This is step taken when enough discussion of the tradeoffs have taken place and the team is in a position to make a decision. Before entering FCP, a super majority of the team must sign off.
+- The FCP will last 7 days. If there's unanimous agreement among the team the FCP can close early.
+- For voting, the binding votes are comprised of the maintainer team. Acceptance requires a super majority of binding votes in favor. The voting options are the following: Affirmative, Negative, and Abstinence. Non-binding votes are of course welcome. Super majority means 2/3 or greater.
+- If no substantial new arguments or ideas are raised, the FCP will follow the outcome decided. If there are substantial new arguments, then the RFC will go back into development.
+
+Once an RFC has been accepted, the team member who merges the pull request should do the following:
+
+- Assign an id based off the latest available number.
+- Rename the file based off the id inside `rfc/`.
+- Fill in the remaining metadata at the top.
+- Commit everything.
+
+### Status Field
+The status field in the meta section has the following acceptable values:
+- **Draft**: The maintainer team has not yet approved the RFC. Edits are still being made.
+- **Approved**: The maintainer team has approved the RFC and is accepting PRs for its implementation.
+- **On Hold**: The maintainer team has approved the RFC, but PRs are currently not being accepted for its implementation. A reason must be provided in the RFC.
+- **Superseded**: The maintainer team has approved the RFC, but it is replaced by another RFC.
+
+RFCs that are rejected will no longer be pulled into the `rfc/` directory.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Though RFCs are intended to be lightweight, this introduces extra process than what we're doing today. The additional work may make consistency hard.
+
```

Comment:
An additional drawback:
In not archiving rejected/unapproved RFCs, Carto may lose context of past discussions and decisions. Sometimes a decision not to do X can be as significant as the decision to do Y.

Reply:
That's fair. I think those could be found by looking at past PRs with the `rfc` label so we wouldn't lose that context but I hear it's a little less visible.